src/Domain/Person.php

```diff
@@ -68,6 +64,22 @@ class Person
      */
     public array $cash_balance = [];
 
+    /**
+     * @OA\Property(
+     *     type="object",
+     *     description="Properties are lowercase currency codes, e.g. 'gbp'. Values are
+     *     available amounts in smallest denomination, e.g. 123 pence.",
+     *     example={
+     *         "eur": 0,
+     *         "gbp": 123,
+     *     }
+     * )
+     * @var null|int[]  Pending Payment Intents for Big Give (i.e. donor fund top up tips)
+     *                  in smallest unit (cents/pence), keyed on lowercase currency code.
+     *                  Or may be null if no tip balances requested on Get.
+     */
+    public ?array $pending_tip_balance = [];
```

Comment:
The docblock says it may be null but it's never set to null - should this be

```suggestion
    public ?array $pending_tip_balance = null;
```

?

Reply:
Yep, good spot. Like you I'm slightly in 2 minds about this being nullable but didn't want to add complexity or avoidable, frequent Stripe callouts. But of course we need to distinguish between "don't know" and "none" so this was incorrect.